app/models/guide.rb

```diff
@@ -18,4 +18,17 @@ class Guide < ActiveRecord::Base
   def work_in_progress_edition?
     latest_edition.try(:state) == 'draft'
   end
+
+  def can_request_review?
+    return false if latest_edition.nil?
+    return false if !latest_edition.persisted?
+    return false if latest_edition.review_requested?
```

Comment:
What if the latest edition has been published, will it be covered by `latest_edition.review_requested?`